### DISCUSSION TITLE
Ultimarc BlueHD

### DISCUSSION THREAD

**Discussion by lamprey13** [darthcloud/BlueRetro#1112] [Q&A]
Just curious if anyone has tried using the Ultimarc BlueHD to connect to a BlueRetro receiver and if it worked?

The product website:
https://www.ultimarc.com/control-interfaces/u-hid-en/bluehid/

I'm looking for a bluetooth "arcade" encoder pcb that works with BlueRetro receivers and supports 21+ buttons/inputs. I know that the BlueRetro receivers don't work with some controllers unless theBlueRetro firmware has the right info (sorry not sure of the right words for this) to know how to map the inputs and all that.

At any rate,  If anyone knows of anything that meets that criteria and works with BlueRetro, please let me know!

Cheers!

**Comment by darthcloud** [comment]
Hard to tell someone will have to try, if it doesnt it should be possible to fix by making a debug trace on blueretro while connecting the controller.
